wled00/wled.cpp

```diff
@@ -221,7 +221,7 @@ void WLED::loop()
 
     yield();
 
-    if (!offMode || strip.isOffRefreshRequred)
+    if (!offMode || strip.isOffRefreshRequred || enforceOff)
       strip.service();
```

Comment:
This will cause strip to be updated if `enforceOff==true` is this the intended behaviour?
How does this enforce "off" state of the strip?
When strip is updated it executes effect functions (including mode_solid()).